### PR TITLE
:sparkles: Add searchbar to emote picker

### PR DESCRIPTION
**Pull request by Daywalkerchen**

### CODE REVIEW

**Comment by Daywalkerchen** [comment]
🥺 May someone review me?

File: src/components/emote-picker/EmotePicker.jsx [hunk 1]
```diff
@@ -3,7 +3,19 @@ import { EMOTES } from '../../const/emotes';
 import Emote from './emote/Emote';
 import './emotePicker.scss';
 
-const EmotePicker = ({ onClick }) => {
+const EmotePicker = ({ searchString, onClick }) => {
+  console.log(searchString);
+
+  if (searchString) {
```
Comment (Absebo) [inline src/components/emote-picker/EmotePicker.jsx:9]: Looks like a bit of a code duplication here.

Both of your if-else-branches are mostly doing the same thing.
```
EMOTES
.filter((emote) => !searchString || emote.tags[0].toLowerCase().includes(searchString.toLowerCase())).`
.map(...)
```

File: src/components/emote-picker/EmotePicker.jsx [hunk 2]
```diff
@@ -3,7 +3,19 @@ import { EMOTES } from '../../const/emotes';
 import Emote from './emote/Emote';
 import './emotePicker.scss';
 
-const EmotePicker = ({ onClick }) => {
+const EmotePicker = ({ searchString, onClick }) => {
+  console.log(searchString);
```
Comment (Absebo) [inline src/components/emote-picker/EmotePicker.jsx:7]: please remove `console.log`

File: src/components/search-bar/SearchBar.jsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+  useEffect(() => {
+    onSetSearchString(searchString);
+  }, [searchString]);
+
+  return (
+    <div className="search-bar">
+      <input
+        type="text"
+        placeholder="Suche"
```
Comment (Absebo) [inline src/components/search-bar/SearchBar.jsx:15]: > Suche

Should this really be a german word?

**Review by Absebo — CHANGES_REQUESTED** [review]

**Review by cecob — APPROVED** [review]
lgtm

**Review by Absebo — APPROVED** [review]

**Pull request merged**